pkg/test/resourcefixture/testdata/basic/datacatalog/v1alpha1/datacatalogentrygroup/datacatalogentrygroup-minimal/_generated_object_datacatalogentrygroup-minimal.golden.yaml

```diff
@@ -4,27 +4,27 @@ metadata:
   finalizers:
   - cnrm.cloud.google.com/finalizer
   - cnrm.cloud.google.com/deletion-defender
-  generation: 2
+  generation: 1
   labels:
     cnrm-test: "true"
-  name: datacatalogentrygroup-minimal-${uniqueId}
+  name: datacatalogentrygroup-minimal${uniqueId}
   namespace: ${uniqueId}
 spec:
-  displayName: Updated Minimal Entry Group
+  displayName: Initial Minimal Entry Group
```

Comment:
Not sure why the golden output does not reflect the changes made by the update.yaml.

Reply:
You are looking at an outdated file ? The latest file has Updated.

Reply:
This was in the commit that "captures the real GCP log". Later it is override by the commit that "captures the mock GCP log".